app/src/main/java/at/roteskreuz/stopcorona/model/entities/infection/info/ApiInfectionInfoRequest.kt

```diff
@@ -8,15 +8,15 @@ import com.squareup.moshi.ToJson
 import org.threeten.bp.LocalDate
 import org.threeten.bp.format.DateTimeFormatter
 
-enum class WarningType {
+enum class WarningType(val transmissionRiskLevel: Int) {
     @field:Json(name = "yellow-warning")
-    YELLOW,
+    YELLOW(5),
```

Comment:
This is the risk mapping as per the specification and Exposure Notification Framework. It will not change.
I will link to the documentation to explain why these "magic numbers" exist.